docs/references/nextjs/auth.mdx

```diff
@@ -32,54 +65,72 @@ import { auth } from '@clerk/nextjs';
+  
   const data = { supabaseData: 'Hello World' };
+  
   return NextResponse.json({ data });
 }
 ```
 
-## Authorization
+## Use `auth()` to check if a user is authorized
 
-Use `has` in order to check if a user is authorized to access a component.
+`auth()` returns the `Authentication` object, which includes the `has()` helper. `auth()` also returns the `protect()` helper. 
 
-```tsx filename="app/page.[jsx/tsx]"
+`has()` and `protect()` can be used to check if a user is authorized to access certain parts of your application.
+
+### Use `has()` to check if a user is authorized
+
+Use the [`has()`](/docs/references/nextjs/authentication-object#has) helper in order to check if a user is authorized to access a component.
+
+In the following example, the `has()` helper is used to check if a user has the `org:team_settings:manage` permission. If the user does not have the permission, `null` is returned. If the user has the permission, the component will return the "Team Settings" heading.
+
+```tsx filename="app/page.tsx"
 import { auth } from '@clerk/nextjs';
 
 export default async function Page() {
   const { has } = auth();
 
-  const canManage = has({permission:"org:team_settings:manage"});
+  const canManage = has({ permission:"org:team_settings:manage" });
 
   if(!canManage) return null;  
 
-  return <h1v>Team Settings</h1>
+  return <h1>Team Settings</h1>
 }
 ```
 
-## Authorization with `protect`
+### Use `protect()` to check if a user is authorized
+
+Use the [`protect()`](#protect) helper to protect a route handler from unauthorized access.
 
-<Callout type="warning" emoji="⚠️">
+<Callout type="warning">
   `auth().protect()` only works for App Router and is considered experimental.
 </Callout>
 
-```tsx filename="app/api/route.[js/ts]"
+In the following example, the `protect()` helper is used to check if a user is authorized to access a route handler. If the user is not authorized, the `protect()` helper will throw a `notFound` Next.js error. If the user is authorized, the `protect()` helper will return the `Authentication` object, which has the `userId` property.
+
+```tsx filename="app/api/route.ts"
 import { auth } from "@clerk/nextjs";
 
 export const POST = () => {
-  const { userId } = auth().protect({permission: "org:team_settings:manage"});
+  const { userId } = auth().protect({ permission: "org:team_settings:manage" });
 
   return users.createTeam(userId);
 }
 ```
 
+## Use `auth()` to check your current user's role
 
-## Check your current user's role
+In some cases, you need to check your user's current organization role before displaying data or allowing certain actions to be performed. 
 
-In some cases, you need to check your user's current organization role before displaying data or allowing certain actions to be performed.
+You can use the `orgRole` property of the `Authentication` object, which is returned by the `auth()` helper, to check the user's current role.
```

Comment:
```suggestion
Check the current user's role with the `orgRole` property of the `Authentication` object returned by `auth()`, as shown in the following example:
```

Puts the purpose of the action at the front of the sentence; mentions that it's demonstrated in the below example